src/readBlacklist.ts

```diff
@@ -20,6 +20,7 @@ export const readBlacklist = new Set([
   "vectorNetwork",
   "masterComponent",
   // Figma exposes this but plugin types don't support them yet
+  "selection",
```

Comment:
Existing comment above makes it seem like this is commented out because plugin types don't support it yet (incorrect)